docs/development/local-development.md

```diff
@@ -136,23 +128,24 @@ If this is working then in future you can create other test repos to verify your
 ## Tests
 
 You can run `yarn test` locally to test your code.
-We test all PRs using the same tests, run on CircleCI and Azure Pipelines. `yarn test` runs an `eslint` check, a `prettier` check, a `type` check and then all the unit tests using `jest`.
-
-## Jest
-
-You can run just the Jest unit tests by running `yarn jest`.
-You can also run just a subset of the Jest tests using file matching, e.g. `yarn jest composer` or `yarn jest workers/branch`.
-If you get a test failure due to a "snapshot" mismatch, and you are sure that you need to update the snapshot, then you can append `-u` to the end.
-e.g. `yarn jest composer -u` would update the saved snapshots for _all_ tests in `test/manager/composer/*`.
+We test all PRs using the same tests, run on GitHub Actions.
+`yarn test` runs an `eslint` check, a `prettier` check, a `type` check and then all the unit tests using `jest`.
 
-#### Prerequisites
+### Prerequisites
 
 You need to have Python with `mock` installed for all tests to pass.
 Python 3 includes `mock` so that approach is recommended.
```

Comment:
I think this can be removed, as python 3.7 has that buildin and we force >=3.7

Maybe the next sentense can be mored somewhere else and we remove this section completely

Reply:
I've moved the sentence about the `.npmrc` file up into the fork and clone steps so that it's more visible.